Backpack-SwiftUI/Slider/Classes/SliderThumbView.swift

```diff
@@ -34,12 +35,15 @@ struct SliderThumbView: View {
                     .onChanged { value in
                         onDrag(value)
                     }
+                    .onEnded { _ in
```

Comment:
is this also called when using the accessibility controls?

Reply:
good catch! no, it would not, i'll call the event when the accessibiltiy methods are executed!